components/Ripple/docs/typical-use.md

```diff
@@ -253,14 +253,14 @@ MDCStatefulRippleView *statefulRippleView = [[MDCStatefulRippleView alloc] init]
   [statefulRippleView touchesEnded:touches withEvent:event];
   [super touchesEnded:touches withEvent:event];
 
-  statefulRippleView.rippleHighlighted = NO; 
+  statefulRippleView.rippleHighlighted = NO;
 }
 
 - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {
   [statefulRippleView touchesCancelled:touches withEvent:event];
   [super touchesCancelled:touches withEvent:event];
 
-  statefulRippleView.rippleHighlighted = NO; 
+  statefulRippleView.rippleHighlighted = NO;
 }
 ```
-<!--</div>-->
\ No newline at end of file
+<!--</div>-->
```

Comment:
Please re-run the docs generator for this component.

Reply:
Done